batfish.config.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const path = require('path');
+const navigationStructure = require('./_tmp_assembly/navigation.json');
+
+module.exports = () => {
+  return {
+    dataSelectors: {
+      examplesSubNavigationList: () => {
+        return navigationStructure
+          .navigationList
+          .find((routeConfig) => routeConfig.name === 'Examples')
+          .items;
+      },
+      documentationData: () => {
+        return navigationStructure
+          .navigationList
+          .find((routeConfig) => routeConfig.name === 'Documentation' )
+          .props
+          .documentationData;
+      }
+    },
+    fileLoaderExtensions: (extensions) => extensions.concat(['svg', 'ico']),
```

Comment:
I think the Favicon should be added directly to the root of the pages directory. It should be copied from there to the same place in the built files.